netclient/bin-maker.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+if [[ ${#VERSION} -lt 1 ]]; then VERSION="develop"; fi 
```

Comment:
replace lines 1-2 with 
echo "build with version tag '${VERSION:-develop}'"

Reply:
That doesn't update the value of VERSION in the case where it is not set.

Reply:
Checks if unset, setting it otherwise. 
```bash
if [[ ${#VERSION} -lt 1 ]]; then VERSION="develop"; fi  
```
relevant for..
```bash
-ldflags="-X 'main.version=$VERSION'"
```